src/js/messaging/containers/Thread.jsx

```diff
@@ -160,6 +172,7 @@ class Thread extends React.Component {
 const mapStateToProps = (state) => {
   return {
     charsRemaining: state.messages.ui.charsRemaining,
+    folders: state.folders,
```

Comment:
Expand it out to `state.folders.data.items` so you can just pass `this.props.folders` in the `render`. Idea being that `mapStateToProps` can act as the bridge so that the component won't have to navigate through the specific structure of the store itself.